File: app/src/main/kotlin/com/moviereel/data/api/model/tv/Seasons.java
package com.moviereel.data.api.model.tv;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author lusinabrian on 10/04/17
 */

public class Seasons {

    @Expose
    @SerializedName("air_date")
    private String airDate;

    @Expose
    @SerializedName("episode_count")
    private int episodeCount;

    @Expose
    @SerializedName("id")
    private int id;

    @Expose
    @SerializedName("poster_path")
    private String posterPath;

    @Expose
    @SerializedName("season_number")
    private int seasonNumber;
}

File: remote/src/main/kotlin/com/moviereel/remote/models/tv/Seasons.kt
package com.moviereel.remote.models.tv

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author lusinabrian on 10/04/17
 */

data class Seasons(

        @Expose
        @SerializedName("air_date")
        private val airDate: String? = null,

        @Expose
        @SerializedName("episode_count")
        private val episodeCount: Int = 0,

        @Expose
        @SerializedName("id")
        private val id: Int = 0,

        @Expose
        @SerializedName("poster_path")
        private val posterPath: String? = null,

        @Expose
        @SerializedName("season_number")
        private val seasonNumber: Int = 0
)
